contracts/minter-suite/Minters/MinterMerkle/MinterMerkleV2.sol

```diff
@@ -51,8 +51,7 @@ contract MinterMerkleV2 is ReentrancyGuard, IFilteredMinterMerkleV0 {
     using MerkleProof for bytes32[];
 
     /// Delegation registry address
-    address public constant DELEGATION_REGISTRY_ADDRESS =
-        0x00000000000076A84feF008CDAbe6409d2FE638B;
+    address public immutable DELEGATION_REGISTRY_ADDRESS;
```

Comment:
we opted to keep ALL_CAPS_CASING here to remain compatible with the existing, deployed MinterMerkleV2 contracts. If we don't like this, we should roll to a MitnerMerkleV3 in this PR

Reply:
No strong opinion here on casing personally. If we don't typically do it for other immutables, we probably don't need to do it here.

Reply:
I've updated this to be our typical casing for immutables, especially since we rolled these updates into a new version of the minter.